src/Core/OrganizationFeatures/OrgUser/Invitation/Accept/IOrganizationUserAcceptCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
The commands seem more like single-method services to me. It's good to break functionality out `OrganizationService`, but I think this _may_ be too far in the opposite direction - everything feels really compartmentalised. (A lot of additional folder hierarchy, interfaces, classes, etc.) My assumption was that you'd just create an `OrganizationUserService` which contains all of these methods. What's the benefit of this command pattern here compared to just creating another service? Where do "command" classes fit in in the existing hierarchy of controller -> service -> repository? (Note that I may be too biased in favour of our current patterns)